juntagrico/entity/textoverride.py

```diff
@@ -0,0 +1,39 @@
+from pathlib import Path
+
+from django.db import models
+from django.template.loader import get_template
+from django.utils.translation import gettext as _
+
+from juntagrico.entity import JuntagricoBaseModel
+
+
+def get_overridable_templates():
+    """ collets the overridable templates from the textsnippets folder
+    :return: an iterable of tuples with the relative paths to overridable text templates and its file name
+    """
+    local_templates = Path(__file__).parent.parent / 'templates'
```

Comment:
to check is there a method ot get all templates available from django and filter those for a specific juntagrico prefix in template name?
advantage would be that we do not rely on the file system and also overide snippets can be in different folder in the future without touching this piece if code